docs/sdks/server-sdks/dotnet/quickstart.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Eppo .NET SDK enables feature flags and experiments in your .NET applications with only a few lines of code.
+
+The SDK handles all the complexity of feature flag evaluation and experiment assignment locally in your application, with no network calls required after initial setup. This guide will walk you through installing the SDK and implementing your first feature flag, experiment, and contextual bandit.
```

Comment:
This SDK is primarily a server SDK and will poll for updated configuration every 30sec (configurable), so there are actually continuous network calls.

Reply:
Understood. I think I didn't understand that the only SDKs that really benefit from the offline mode use cases were clients.